src/source-dpdk.c

```diff
@@ -267,17 +268,18 @@ void TmModuleDecodeDPDKRegister(void)
 
 static inline void DPDKDumpCounters(DPDKThreadVars *ptv)
 {
-    struct rte_eth_stats eth_stats;
-    int retval = rte_eth_stats_get(ptv->port_id, &eth_stats);
-    if (unlikely(retval != 0)) {
-        SCLogError("Failed to get stats for port id %d: %s", ptv->port_id, rte_strerror(-retval));
-        return;
-    }
-
     /* Some NICs (e.g. Intel) do not support queue statistics and the drops can be fetched only on
      * the port level. Therefore setting it to the first worker to have at least continuous update
      * on the dropped packets. */
     if (ptv->queue_id == 0) {
+        struct rte_eth_stats eth_stats;
+        int retval = rte_eth_stats_get(ptv->port_id, &eth_stats);
+        if (unlikely(retval != 0)) {
+            SCLogError(
+                    "Failed to get stats for port id %d: %s", ptv->port_id, rte_strerror(-retval));
```

Comment:
can we prefix the message with the iface?

Reply:
Prefixed.